### PR TITLE
emjshrx: Week 1 Solution

### PR DESCRIPTION
**Pull request by emjshrx**
Creating an early PR since I will be travelling over the weekend and wont get time to work on this. 

Have done the exercises quite plainly except for the last one of  pretty printing all transaction details. Will work on it if I get the time.

### CODE REVIEW

**Comment by rajarshimaitra** [comment]
Superb. Looks really good. Will address a few comments once I try it out.

File: exercises/week 1/solutions/emjshrx.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+txindex=1" >> ~/.bitcoin/bitcoin.conf
+    bitcoind -daemon
+    sleep 5
+    bitcoin-cli createwallet "Miner"
+    bitcoin-cli createwallet "Trader"
+    bitcoin-cli loadwallet Miner
+    mineraddr=$(bitcoin-cli -rpcwallet=Miner getnewaddress "Mining Reward")
+    bitcoin-cli generatetoaddress 101 $mineraddr
+    # coinbase transactions are locked for 100 blocks to accomadate reorgs in this time
+    bitcoin-cli getbalance
```
Comment (rajarshimaitra) [inline exercises/week 1/solutions/emjshrx.sh:45]: This needs an `-rpcwallet=Miner`

**Review by rajarshimaitra — COMMENTED** [review]
ACK one modification needed to make the script pass.

File: exercises/week 1/solutions/emjshrx.sh (hunk 1 above)
Comment (emjshrx) [reply exercises/week 1/solutions/emjshrx.sh]: fixed. yes missed it while debugging

**Pull request merged**